edgelet/iotedge/src/check/checks/iotedged_version.rs

```diff
@@ -0,0 +1,95 @@
+use std::process::Command;
+
+use failure::{self, Context, Fail, ResultExt};
+use regex::Regex;
+
+use crate::check::{checker::Checker, Check, CheckResult};
+
+#[derive(Default, serde_derive::Serialize)]
+pub(crate) struct IotedgedVersion {
+    version: Option<String>,
```

Comment:
Rename this to `actual_version` when you add `expected_version` per my comment below.